src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/view/fragments/SelectCollection.kt

```diff
@@ -79,6 +85,15 @@ class SelectCollection : View() {
             else -> MaterialIconView(MaterialIcon.COLLECTIONS_BOOKMARK, "50px")
         }
     }
+
+    private fun doesProjectExist(projectList: List<ProjectCollection>, thisCollection: Collection): Boolean {
+        for (project in projectList) {
+            if (project.titleKey == (thisCollection.titleKey)) {
+                return true
+            }
+        }
+        return false
```

Comment:
You might be able to rewrite this in a more functional way to avoid the for loop
```kotlin
return projectList.map { it.titleKey }.contains(thisCollection.titleKey)
```

Reply:
yes, I wanted to do that, just was unsure as to how. thanks